app/src/common/polyfills.js

```diff
@@ -21,7 +21,6 @@ import 'array.findindex';
 import 'promise-polyfill/src/polyfill';
 import 'polyfill-array-includes';
```

Comment:
What about this imports and above?
Also we have it in the package.json file too